core/src/test/java/kafka/test/api/ShareConsumerTest.java

```diff
@@ -1922,6 +1918,38 @@ private void warmup() throws InterruptedException {
         }
     }
 
+    private void verifyShareGroupStateTopicRecordsProduced() {
+        try {
+            TestUtils.waitForCondition(() ->
+                    !cluster.brokers().get(0).metadataCache().getAliveBrokerNodes(new ListenerName("EXTERNAL")).isEmpty(),
+                DEFAULT_MAX_WAIT_MS, 100L, () -> "cache not up yet");
+            Map<String, Object> consumerConfigs = new HashMap<>();
+            consumerConfigs.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, cluster.bootstrapServers());
+            consumerConfigs.put(ConsumerConfig.GROUP_ID_CONFIG, UUID.randomUUID().toString());
```

Comment:
I think you can remove the group.id config too because of the use of KafkaConsumer.assign.